src/Symfony/Component/Workflow/README.md

```diff
@@ -4,7 +4,7 @@ Workflow Component
 Resources
 ---------
 
-  * [Documentation](https://symfony.com/doc/current/components/workflow/introduction.html)
+  * [Documentation](http://symfony.com/doc/master/components/workflow.html)
```

Comment:
Sorry, that was not my intention. I just copied the link from the address bar. 

I guess there was no http://symfony.com/ => https://symfony.com/ redirect.